polyfills/Array/prototype/@@iterator/polyfill.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+
+// from https://github.com/medikoo/es6-iterator/blob/master/array.js
+'use strict';
+
+var setPrototypeOf = require('es5-ext/object/set-prototype-of')
+  , contains       = require('es5-ext/string/#/contains')
+  , d              = require('d')
+  , Iterator       = require('./')
+
+  , defineProperty = Object.defineProperty
+  , ArrayIterator;
```

Comment:
If you're importing this from a third party source then fine (but you need to add `repo` and `licence` props to the config), but if you're authoring, then one var dec per line please :-)